### DISCUSSION TITLE
5MW Semisubmersible OC4 DeepWind Model

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1774] [Q&A]
Hello, everyone, where can I download the 5MW Semisubmersible OC4 DeepWind Model？

**Comment by jjonkman** [comment]
Dear @sjihf12,

 An OpenFAST model of the NREL 5-MW baseline wind turbine atop the OC4-DeepCwind semisubmersible is available through the OpenFAST r-test: https://github.com/OpenFAST/r-test/tree/main/glue-codes/openfast/5MW_OC4Semi_WSt_WavesWN.

Best regards,

**Reply by sjihf12** [reply]
Thank you!

**Comment by Tetsuya2000** [comment]
Hi is there a 10MW version of this? @jjonkman

**Comment by jjonkman** [comment]
Dear @Tetsuya2000,

IEA Wind Task 37 developed a 10-MW reference wind turbine model based on the DTU 10-MW reference turbine, but I'm only aware that fixed-bottom support structure with monopile substructure is available for this turbine: https://github.com/IEAWindTask37/IEA-10.0-198-RWT.

The following forum topic on the DTU 10-MW reference wind turbine may also interest you: https://forums.nrel.gov/t/dtu-10-mw-reference-wind-turbine/993.

Best regards,
